themes/bootstrap3/templates/search/versions.phtml

```diff
@@ -0,0 +1,108 @@
+<?php
+  // Set up page title:
+  $lookfor = $this->results->getUrlQuery()->isQuerySuppressed() ? '' : $this->params->getDisplayQuery();
+  $headTitle = $this->slot('head-title')->get($this->translate('Versions'));
+  $this->headTitle($headTitle);
+
+  $this->slot('search-heading')->set('');
```

Comment:
What is the purpose of this, and the associated refactoring of the reading of the slot in showing.phtml? It's not immediately obvious to me; just checking to see if this is an artifact of an earlier approach, or if it's important but perhaps needs a comment.

Reply:
The change in showing.phtml avoids a space before the comma in `, query time` when search-heading is set to an empty string. And it's set to an empty string to avoid `for search '',`. Do you think it would be better to not use showing.phtml at all here?

Reply:
That makes sense to me. Might be worth a comment in showing.phtml so nobody changes it back without thinking it through. As to whether to use showing.phtml at all here, I'll give that some more thought next time I have time for my next hands-on test.

Reply:
Comment added.